BraintreeCore/src/main/java/com/braintreepayments/api/AuthorizationProvider.java

```diff
@@ -6,7 +6,7 @@
  * Implement this interface to provide an asynchronous way for {@link BraintreeClient} to fetch
  * a client token from your server when it is needed.
  */
-public interface AuthorizationProvider {
+interface AuthorizationProvider {
```

Comment:
Is this ok that it's no longer public?

Reply:
Yeah we're still back and forth on the naming. Going to add back soon 🤞